packages/profile/src/views/Profile.jsx

```diff
@@ -8,11 +8,21 @@
 
 import { useEffect, useReducer } from 'react'
 import { useNavigate } from 'react-router-dom'
+import Resizer from 'react-image-file-resizer'
 import { Switch } from '@headlessui/react'
-import { useServices, Navbar, Footer, Alert } from '@kernel/common'
+import { useServices, Navbar, Footer, Alert, getUrl } from '@kernel/common'
 
 import AppConfig from 'App.config'
 
+const AVATAR_CONFIG = {
```

Comment:
How does this ensure the size stays below a limit?

Reply:
If you mean the file size, it doesn't, and neither would a straightforward canvas implementation. I figure there is an upper bound to how large of a file a 500x500 image can be, and resizing it by height/width would keep it reasonably small in the vast majority of cases (or maybe in all cases if there's not some way that a 500x500 image can be huge). Since you didn't seem to be attached to a specific hard limit, it didn't feel necessary to sink more effort into enforcing both a max height/width and a max file size.

Reply:
File size is my primary concern and it should definitely be checked.